src/GeoJson.php

```diff
@@ -23,6 +23,16 @@
  */
 abstract class GeoJson implements JsonSerializable, JsonUnserializable
 {
+    public const TYPE_LINESTRING = 'LineString';
```

Comment:
```suggestion
    public const TYPE_LINE_STRING = 'LineString';
```

Adding an underscore for consistency with `TYPE_MULTI_LINE_STRING` and types with camelCase.